sensor_msgs/PointCloud2[] array